insights/parsers/pmrep.py

```diff
@@ -0,0 +1,80 @@
+"""
+Pmrep - command ``pmrep -t 1s -T 1s network.interface.out.packets network.interface.collisions swap.pagesout``
+==============================================================================================================
+
+Parse the content of the ``pmrep -t 1s -T 1s network.interface.out.packets network.interface.collisions swap.pagesout`` command.
+
+Sample ``pmrep -t 1s -T 1s network.interface.out.packets network.interface.collisions swap.pagesout`` command output::
+
+    n.i.o.packets  n.i.o.packets  n.i.collisions  n.i.collisions  s.pagesout
+             eth0             lo            eth0              lo
+          count/s        count/s         count/s         count/s     count/s
+              N/A            N/A             N/A             N/A         N/A
+            1.997          0.000           0.000           0.000       0.000
+
+Examples:
+    >>> type(pmrep_doc_obj)
```

Comment:
Please use the test data in the above sample in the exmples.

Reply:
Updated the test data.